site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -59,55 +57,58 @@ Now that we have moved state management to a dedicated module, we need a way to
 
 ## Injectors
 
-{% task 'Register an `Injector` that will allow state to be injected into widgets.' %}
+{% task 'Register an injector factory that will allow state to be injected into widgets.' %}
 
 Currently, the application's `main` module is only responsible for creating the `Projector`, which provides the bridge between the application code and the DOM.
 
 {% include_codefile 'demo/initial/biz-e-corp/src/main.ts' %}
 
 Now, we need to:
 
-1. Update the `main` module to construct and populate an instance of `ApplicationContext`
-2. Create a `registry` and then define the `ApplicationContext` as an injector in the registry
-3. To make the `registry` available within the widget tree, we need to pass the `registry` as a property to the `projector`
+1. Create a `registry` and then define an injector factory that creates the `ApplicationContext` passing the invalidator and initial state. Returning a function that returns this instance.
```

Comment:
Maybe expand on "Returning a function that returns this instance.". E.g. "The injector factory returns a function that returns the `ApplicationContext`."

Reply:
Done